src/components/app.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import ComparativeD3Graph from './comparativeD3Graph'
+import Nav from './nav'
+import SelectYear from './selectYear'
+import HistoryData from './historyData'
+import CurrentData from './currentData'
+import reactCSS from 'reactcss'
+
+const styles = reactCSS({
+  default: {
+    app: {
+      	height: '100%',
+      	textAlign: 'center',
+      	color: 'white',
+      	paddingBottom: 0,
+    },
+    title: {
+      	paddingTop: 25,
+      	marginTop: 0,
+      	paddingBottom: 25,
+      	borderBottom: '1 #0e4b51 solid'
+    }
+  }
+})
+
+
+const App = () => {
```

Comment:
Nice refactor.

You can take it even one step further. You don't need the ` { return (<stuff/>) }`. E.g.:

```js
const App = () => (
  <div style={styles.app} className="App">
    <Nav />
    <ComparativeD3Graph />
    <SelectYear />
    <CurrentData /><HistoryData />
  </div>
)
```